NUnitLite-1.0.0/src/framework/FinallyDelegate.cs

```diff
@@ -70,11 +70,12 @@ public class FinallyDelegate
 		public void Set (TestExecutionContext context, long startTicks, TestResult result) {
 			var frame = new Tuple<TestExecutionContext, long, TestResult>(context, startTicks, result);
 
+			var name = result.FullName + "_" + result.Test.FullName + "_" + result.GetHashCode ();
```

Comment:
`result.FullName` and `result.Test.FullName` have the same value for parameterized tests (and it correctly includes the parameter value).

Additionally you're not changing the retrieval logic in `HandleUnhandledExc ()` so this would fail when an exception does occur.